sys/include/net/gnrc/netif/hdr.h

```diff
@@ -87,6 +87,7 @@ typedef struct {
     uint8_t flags;              /**< flags as defined above */
     int16_t rssi;               /**< rssi of received packet in dBm (optional) */
     uint8_t lqi;                /**< lqi of received packet (optional) */
+    uint8_t seq;                /**< seq of received packet (optional) */
```

Comment:
If this is really required (but since this shouldn't something the network layer needs to deal with I'm not sure it is): `#ifdef MODULE_GNRC_GOMACH` around